mlrun/projects/project.py

```diff
@@ -3244,7 +3246,16 @@ def set_model_monitoring_credentials(
                 "stream_path": stream_path,
                 "tsdb_connection": tsdb_connection,
             },
+            force=force,
         )
+        if force:
+            logger.info(
+                "Model monitoring credentials were set successfully. "
+                "Please take in mind that if you already had model monitoring functions "
+                "/ model monitoring infra deployed on your project, you will need to redeploy them."
+                "For redeploying the model monitoring infra, please use the `enable_model_monitoring` Api "
+                "and set `rebuild_images=True`"
```

Comment:
```suggestion
                "Model monitoring credentials were set successfully. "
                "Please keep in mind that if you already had model monitoring functions "
                "/ model monitoring infra deployed on your project, you will need to redeploy them."
                "For redeploying the model monitoring infra, please use `enable_model_monitoring` API "
                "and set `rebuild_images=True`"
```

Reply:
In addition, I'll add this comment to the docstring as well. I think it's important that the user will understand what `force` does before calling it because it can be a "killing" action that cannot be undone.